build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,12 +71,6 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
-                    AzurePublisherTool.ListBlobs($"{targetContainer}")
```

Comment:
The issue is we're deleting all the binaries here as well, so we need to match against file patterns that match a version, I think I'll use a regex from updatedependencies.